### PR TITLE
Sticky modal close area

### PR DESCRIPTION
**Pull request by mperrotti**
#### Related issues
Fixes https://meetup.atlassian.net/browse/SDS-672

#### Description
Added a prop to `Modal` that allows us to make the close area of a modal stick to the top of the modal

#### Screenshots (if applicable)
modalstickyheader

### CODE REVIEW

**Comment by akdetrick** [comment]
this is a new feature introduced as a non-breaking change; would be good to bump the minor version and changelog for this

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# [4.11]
+
+- **Change** Added the ability to set the close area at the top of a Modal to be sticky using the `stickyCloseArea` prop
```
Comment (akdetrick) [inline CHANGELOG.md]: we can be more specific here and call the change out as `**Feature**`

File: assets/scss/components/_modal.scss [hunk 2]
```diff
@@ -73,6 +72,18 @@ $modal-width: 440px;
 	z-index: map-get($zindex-map, "popup") + 1;
 }
 
+.modal-closeButtonContainer--sticky {
+	background-color: $C_contentBG;
+	border-bottom: 1px solid $C_border;
+	position: sticky;
+	position: -webkit-sticky;
```
Comment (akdetrick) [inline assets/scss/components/_modal.scss]: Should the vendor-prefixed value come _before_ the standard in this case?

Comment (mperrotti) [reply assets/scss/components/_modal.scss]: I think both work, but putting it before feels better in this case

**Pull request merged**